enterprise/backend/test/metabase/embed/settings_test.clj

```diff
@@ -5,41 +5,37 @@
    [metabase.analytics.snowplow-test :as snowplow-test]
    [metabase.embed.settings :as embed.settings]
    [metabase.test :as mt]
-   [metabase.util.log.capture :as log.capture]
    [toucan2.core :as t2]))
 
+(defn- embedding-event?
```

Comment:
Nit: impl/name are checking whether it's an embedding event, docstring says it's removing the embedding event.

Reply:
Fixed